core-addon/DataCollection.js

```diff
@@ -37,6 +37,7 @@ export default class DataCollection {
     // consented to join Rally. Upload is always enabled unless the web-extension
     // is uninstalled.
     Glean.initialize("rally-core", userEnrolled, {
+        appDisplayVersion: browser.runtime.getManifest().version,
```

Comment:
I think this line caused a new unit test failure :( 
```
1) Core
       "before each" hook in "Core":
     Error: done() called multiple times in hook <Core "before each" hook in "Core"> of file /home/circleci/project/tests/core-addon/unit/Core.test.js; in addition, done() received error: TypeError: Cannot read property 'version' of undefined
    at DataCollection.initialize (file:///home/circleci/project/core-addon/DataCollection.js:40:57)
    at file:///home/circleci/project/core-addon/Core.js:38:28
    at <anonymous> {
  uncaught: true
}
      at process._fatalException (internal/process/execution.js:163:25)
      at processPromiseRejections (internal/process/promises.js:213:9)
      at processTicksAndRejections (internal/process/task_queues.js:94:32)
```

Specifically:
`TypeError: Cannot read property 'version' of undefined`

Is this all just mocked out in the unit tests? I know for integration we build a real `manifest.json` but just mocking out that call is probably just fine here.

Reply:
I think `chrome.runtime.getManifest.returns({ version: FAKE_STUDY_VERSION })` should be enough, but I don't see a simple way to test that `FAKE_STUDY_VERSION` matches.

Maybe the version number could be validated and `DataCollection.js` could throw if it fails?

Reply:
> Maybe the version number could be validated and `DataCollection.js` could throw if it fails?

We'd be testing the WebExtension APIs and the Glean APIs by doing this, since there's no specific code from the core add-on that does the collection.

Moreover, doing this kind of test would require us to inspect the Glean payload, which is a discouraged practice for a series of other reasons. I'd recommend leaving this as it is, since we do have tests in Glean.js that verify that configuration values provided at initialization make it into pings.

@rhelmer what do you think?